ko4j/src/main/java/org/netbeans/html/ko4j/Knockout.java

```diff
@@ -315,8 +307,25 @@ function write(val) {
             };
             var cmpt = ko['computed'](bnd);
             cmpt['valueHasMutated'] = function(val) {
-              if (arguments.length === 1) activeGetter = function() { return val; };
-              trigger(val);
+              if (arguments.length === 1) {
+                value = val;
+                trigger(val);
+              } else {
+                var self = ret['ko4j'];
+                if (self) {
+                  var futureValue = self.@org.netbeans.html.ko4j.Knockout::getValue(I)(index);
+                  Promise.resolve(futureValue).then(function(val, e) {
```

Comment:
The whole `ko4j` bindings are now ready for `wait4java=false`, however, that means various updates may happen asynchronously. To get ready for that I had to modify the _Test Compatibility Kit_ to work in _quantums_: https://github.com/apache/netbeans-html4j/commit/1e398f86e740ba4aebef9457e0bea2dadf651b56